connector-packager/connector_packager/jar_jdk_packager.py

```diff
@@ -57,6 +58,15 @@ def get_min_support_version(file_list: List[ConnectorFile], cur_min_version_tabl
             if 2021.1 > float(min_version_tableau):
                 min_version_tableau = "2021.1"
             reasons.append("Connector uses OAuth, which was added in the 2021.1 release")
+        elif connector_file.file_type == "connection-resolver":
+            # Check to see if we're using inferred connection resolver, which needs 2021.1+
+            tdr_root = ET.parse(input_dir / connector_file.file_name).getroot()
+            attribute_list = tdr_root.find('.//connection-normalizer/required-attributes/attribute-list')
+
+            if not attribute_list:
+                if 2021.1 > float(min_version_tableau):
```

Comment:
We don't use float to represent min_version_tableau, it's stored as a string. We only cast to float for the comparisons here. We have a unit test testing a min_tableau_version with a major and minor version number, so that case is supported by the current design: https://github.com/tableau/connector-plugin-sdk/blob/master/connector-packager/tests/test_jar_packager.py#L100